cds_ils/static_pages/search_guide.html

```diff
@@ -2,53 +2,53 @@
 
 <h3>Simple search (one or multiple terms)</h3>
 
-<p><strong>Example:</strong> <a href="#"><code>dark matter</code></a></p>
+<p><strong>Example:</strong> <a href="/search?page=1&size=21&q=dark%20matter"><code>dark matter</code></a></p>
```

Comment:
I am sorry, last comment I promise :)
Can you please remove the arguments `page` and `size` from all the queries? These are automatically added by the React app.
Basically, this is the right path: `/search?q=dark%20matter`